recipes/mkl-service/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set version = "2.0.1" %}
+{% set buildnumber = 0 %}
+
+package:
+    name: mkl-service
+    version: {{ version }}
+
+source:
+    url: https://github.com/IntelPython/mkl-service/archive/v{{version}}.tar.gz
+    sha256: 1ae3032445f6af07071118e9af19b8cd05c72235fe47f4d214035dfdcd8aabfa
+
+build:
+    number: {{ buildnumber }}
+
+requirements:
+    build:
+      - {{ compiler('c') }}
+    host:
+      - python
+      - setuptools
+      - mkl-devel
+      - cython
+      - numpy 1.11.*
+      - six
+    run:
+      - python
+      - mkl
+      - six
+      - {{ pin_compatible('numpy') }}
+
+test:
+    commands:
+      - nosetests -v mkl
+    requires:
+      - nose
+    imports:
+      - mkl
+
+about:
+    home: http://github.com/IntelPython/mkl-service
+    license: BSD 3-Clause
```

From https://opensource.org/licenses/alphabetical:
```suggestion
    license: BSD-3-Clause
    license_family: BSD
```